src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFandom.java

```diff
@@ -0,0 +1,100 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.templates.fandom.AdsFandomTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+
+public class TestAdsFandom extends AdsFandomTestTemplate {
+
+  private void getJquery() {
+    driver.executeScript(
+        "    (function () {\n"
+        + "    var s = document.createElement('script');\n"
+        + "    s.type = 'text/javascript';\n"
+        + "    s.async = true;\n"
+        + "    s.src = 'https://code.jquery.com/jquery-2.2.4.min.js';\n"
+        + "    var x = document.getElementsByTagName('script')[0];\n"
+        + "    x.parentNode.insertBefore(s, x);\n"
+        + "    })();"
+    );
+  }
+
+  @Test(
+      dataProviderClass = AdsDataProvider.class,
+      dataProvider = "fandomAdsPage",
+      groups = {"AdsDesktopPresenceFandom", "AdsFandom"}
+
+  )
+  public void adsFandomDesktopArticleAds(String article) throws IOException {
+    String testedPage = urlBuilder.getUrlForFandomPage(article);
+    AdsBaseObject wikiPage = new AdsBaseObject(driver, testedPage);
+    getJquery();
+    wikiPage.verifyFandomDesktopArticleTopLeaderboard();
```

Comment:
Problem is that in source of page there are always "mobile" and "desktop" versions. So I prefer to have explicite way shown what we are testing. But I will add this note to ADEN-3496.